interface-definitions/include/isis/lfa-local.xml.i

```diff
@@ -0,0 +1,205 @@
+<!-- include start from isis/lfa-local.xml.i -->
+<node name="local">
+  <properties>
+    <help>Local loop free alternate options</help>
+  </properties>
+  <children>
+    <node name="load-sharing">
+      <properties>
+        <help>Load share prefixes across multiple backups</help>
+      </properties>
+      <children>
+        <node name="disable">
+          <properties>
+            <help>Disable load sharing</help>
+          </properties>
+          <children>
+            <leafNode name="level-1">
+              <properties>
+                <help>Disable load sharing for level-1 only</help>
+                <valueless/>
+              </properties>
+            </leafNode>
+            <leafNode name="level-2">
+              <properties>
+                <help>Disable load sharing for level-2 only</help>
+                <valueless/>
+              </properties>
+            </leafNode>
+          </children>
+        </node>
+      </children>
+    </node>
+    <node name="priority-limit">
+      <properties>
+        <help>Limit backup computation up to the prefix priority</help>
+      </properties>
+      <children>
+        <node name="medium">
+          <properties>
+            <help>Compute for critical, high, and medium priority prefixes</help>
+          </properties>
+          <children>
+            <leafNode name="level-1">
+              <properties>
+                <help>Set priority-limit for level-1 only</help>
+                <valueless/>
+              </properties>
+            </leafNode>
+            <leafNode name="level-2">
+              <properties>
+                <help>Set priority-limit for level-2 only</help>
+                <valueless/>
+              </properties>
+            </leafNode>
+          </children>
+        </node>
+        <node name="high">
+          <properties>
+            <help>Compute for critical, and high priority prefixes</help>
+          </properties>
+          <children>
+            <leafNode name="level-1">
```

Comment:
This seems to be a candidate for an XML.I include file

Reply:
Made changes. Please let me know if that's what you wanted my good sir :)